gilded-rose-llc-frontend/__tests__/admin.tests.tsx

```diff
@@ -37,7 +42,7 @@ describe('Admin', () => {
   })
 
   it('should render the update button', () => {
```

Comment:
Question: What happens when the button is pressed?

Reply:
A button press will update the items according to their quality and sellin degradation's. We will add a test in this test suite for this use case!